contrib/opencensus-ext-azure/opencensus/ext/azure/common/schedule.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import threading
+import time
+
+
+class PeriodicTask(threading.Thread):
```

Comment:
@c24t given the semantic is different from the PeriodicTask in metrics exporter (here we subtract the elapsed time from the interval), I want to get your "okay" before refactoring them into a common one.

The signature and member names are borrowed from Python built-in `Timer` class for consistency.

Reply:
The semantic change looks good to me, I'd expect users to expect `PeriodicTask` to work this way -- exporting regularly at some interval regardless of the amount of time it takes to export.